pipelines/GenerativeQuestionAnswering_prompt.yaml

```
@@ -0,0 +1,98 @@
+# If you need help with the YAML format, have a look at https://docs.cloud.deepset.ai/docs/create-a-pipeline#create-a-pipeline-using-yaml.
+# This is a friendly editor that helps you create your pipelines with autosuggestions. To use them, press Control + Space on your keyboard.
+# Whenever you need to specify a model, this editor helps you out as well. Just type your Hugging Face organization and a forward slash (/) to see available models.
+
+# This is the Generative Question Answering pipeline for English with a good embedding-based Retriever and Google's Flan-T5 model
+version: '1.13.1'
+name: 'GenerativeQA_promptnode'
+
+# This section defines nodes that you want to use in your pipelines. Each node must have a name and a type. You can also set the node's parameters here.
+# The name is up to you, you can give your component a friendly name. You then use components' names when specifying their order in the pipeline.
+# Type is the class name of the component. 
+components:
+  - name: DocumentStore
+    type: DeepsetCloudDocumentStore # The only supported document store in deepset Cloud
+  - name: Retriever # Selects the most relevant documents from the document store and passes them on to the Reader
+    type: BM25Retriever # Uses a Transformer model to encode the document and the query
+    params:
+      document_store: DocumentStore
+      top_k: 10 # The number of results to return
```

I apologize if this has been discussed already, but I would recommend pushing this down to `top_k=3`. Currently, there is no mechanism to check that the prompt length is not too long for the model. When using `google/flan-t5-large` the max length is 512 tokens by default and if the prompt is longer than this then the prompt is silently truncated I believe.

---

reduced top_k to 3